src/together/resources/finetune.py

```diff
@@ -941,11 +1045,39 @@ async def create(
             hf_output_repo_name=hf_output_repo_name,
         )
 
+        if from_checkpoint is None and from_hf_model is None:
+            price_estimation_result = await self.estimate_price(
+                training_file=training_file,
+                validation_file=validation_file,
+                model=model_name,
+                n_epochs=finetune_request.n_epochs,
+                n_evals=finetune_request.n_evals,
+                training_type="lora" if lora else "full",
+                training_method=training_method,
+            )
+        else:
+            # unsupported case
+            price_estimation_result = FinetunePriceEstimationResponse(
+                estimated_total_price=0.0,
+                allowed_to_proceed=True,
+                estimated_train_token_count=0,
+                estimated_eval_token_count=0,
+                user_limit=0.0,
+            )
```

Comment:
Nit: I would create a separate variable `price_limit_passed` and assign `price_limit_passed = price_estimation_result.allowed_to_proceed` when there is no CFT but just `price_limit_passed = True` in the unsupported case. This would prevent creating a dummy object with arbitrary values across the file